src/webhooks/pubsub/slackScores.test.ts

```diff
@@ -160,13 +182,208 @@ describe('slackScores Tests', function () {
 | null                          |   - (0/0)      |
 | ospo                          |   - (0/0)      |
 └────────────────────────────────────────────────┘\`\`\``,
-            type: 'mrkdwn',
+              type: 'mrkdwn',
+            },
+            type: 'section',
+          },
+        ],
+        channel: 'G6MCDB51U',
+        text: 'Weekly GitHub Team Scores',
+      });
+    });
+  });
+
+  describe('sendDiscussionMetrics tests', () => {
+    it('should not send message if there was no activity from discussions in the last week', async () => {
+      getDiscussionEventsSpy.mockReturnValue({
+        discussions: [],
+        discussionCommenters: [],
+      });
+      expect(postMessageSpy).not.toHaveBeenCalled();
+    });
+
+    it('should send discussion metrics properly for under 5 discussions/users commented', async () => {
+      const discussions = [
+        {
+          title: 'Discussion 1',
+          repository: 'routing-repo',
+          discussion_number: '001',
+          num_comments: 3,
+        },
+        {
+          title: 'Discussion 2',
+          repository: 'routing-repo',
+          discussion_number: '002',
+          num_comments: 2,
+        },
+        {
+          title: 'Discussion 3',
+          repository: 'test-ttt-simple',
+          discussion_number: '003',
+          num_comments: 1,
+        },
+      ];
+      const discussionCommenters = [
+        {
+          username: 'luke_skywalker',
```

Comment:
Missing a golden opportunity to have one of the commenters be chewbacca:

```
Chewbacca, 3 hours ago:
--------
Raaaaa-ra-ra awwww raaaaa!

- Sent from my iPhone 
```

Reply:
you're right 🤦‍♂️